### PR TITLE
Docker kill implemented

### PR DESCRIPTION
**Pull request by Yuki-Inoue**
No options, but better than nothing.
At least, now we can kill a container.

### CODE REVIEW

File: docker-containers.el [hunk 1]
```diff
@@ -218,7 +219,15 @@ Remove the volumes associated with the container when VOLUMES is set."
...
-(docker-containers-create-selection-functions start stop restart pause unpause rm)
+(docker-containers-create-selection-functions
+  start
+  stop
+  restart
+  pause
+  unpause
+  rm
+  kill
+  )
```
Comment (Silex) [inline docker-containers.el]: Dont put ending `)` on a line of its own please (https://www.gnu.org/software/emacs/manual/html_node/elisp/Coding-Conventions.html#Coding-Conventions)

File: docker-containers.el [hunk 2]
```diff
@@ -286,6 +295,12 @@ Remove the volumes associated with the container when VOLUMES is set."
               (?v "Volumes" "-v"))
   :actions  '((?D "Remove" docker-containers-rm-selection)))
 
+(docker-utils-define-popup docker-containers-kill-popup
+  "Popup for kill signaling containers"
+  'docker-containers-popups
+  :man-page "docker-kill"
+  :actions  '((?K "kill" docker-containers-kill-selection)))
+
```
Comment (Silex) [inline docker-containers.el]: Adding the `SIGNAL` option is pretty easy, can you add it? `:options '((?s "Signal" "-s "))` 

Also the docstring is a bit weird.

**Comment by Silex** [comment]
If the changes requested annoys you just tell me, I'll merge and do it myself :wink:

**Comment by Yuki-Inoue** [comment]
@Silex About the second point out, that adding `-s` option and fixing the document, I had my develop environment blown away, and would require sometime for me to fix it...

So, I'd appreciate if you could just merge and fix it.

**Comment by Silex** [comment]
No worries, thanks!

**Pull request merged**